js/planning.js

```diff
@@ -76,6 +76,11 @@ var GLPIPlanning  = {
          $('#'+GLPIPlanning.dom_id).closest('.ui-tabs').width('98%');
       }
 
+      // Hide some days depending on GLPI configuration
+      var all_days = [0, 1, 2, 3, 4, 5, 6];
+      var enabled_days = JSON.parse(CFG_GLPI.planning_work_days);
```

Comment:
You should not have to decode value everywhere you want to use it.
To decode it during configuration loading, you can call `importArrayFromDB()` in `Config::loadLegacyConfiguration()`.
```suggestion
      var enabled_days = CFG_GLPI.planning_work_days;
```

Reply:
Done, didn't know about this `loadLegacyConfiguration` function 👍  .